src/Piral.Blazor.Tools/tasks/ManagePiletTask.cs

```diff
@@ -396,6 +423,7 @@ public override bool Execute()
                     }
 
                     UpdatePackageVersion();
+                    OverwritePackageJson();
```

Comment:
Actually I would advise to not always run this. Some bundlers restart the process or may do additional checks. We either leave this call here and check in the method if the serialized object = the given object (in which case we don't write it out) or we place a hash of the overwrite package json file in the .blazorrc file and make a comparison here.

Both ways work - the former may be simpler to implement and seems reliable, too.

Reply:
like this?